docs/governance/community-guidelines/releases.md

```diff
@@ -0,0 +1,44 @@
+# Releases
+
+This document is a [community guideline].
+
+## Purpose
+
+The document outlines the guidelines of managing releases within the CCC project.
```

Comment:
Should probably use "Common Cloud Controls (CCC)" the first time and the acronym for all the others